File: 171-excel-sheet-column-number/171-excel-sheet-column-number.cpp
class Solution {
public:
    int titleToNumber(string columnTitle) {
        int k = 0;
        for(int i = 0; i < columnTitle.size(); i++){
            k = k + (25 * k) + ((columnTitle[i] - 'A') + 1);
        }
        return k;
    }
};

File: 171-excel-sheet-column-number/171-excel-sheet-column-number.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
var titleToNumber = function(columnTitle) {
    let sum = 0;
    for(let i = 0; i < columnTitle.length; i++){
        sum += sum*25 + (columnTitle[i].charCodeAt(0) - 'A'.charCodeAt(0) + 1)   
    }
    return sum;
};